server/server/server.go

```diff
@@ -203,11 +204,6 @@ func New(params Params) (*Server, error) {
 
 	server.initHandlers()
 
-	if err := app.InitTemplates(); err != nil {
```

Comment:
This was not needed because the app creation initialize the templates if it is needed.